### ISSUE TITLE
Validation performance

### ISSUE THREAD

**Issue by InstantWindy** [saeedizadi/binseg_pytoch#3]
Hello, Can I ask you a question.
when I was doing semantic segmentation ,the training set loss dropped quickly and training accuracy and Mean IOU also rose, but the validation set loss on just started to decline, but the validation set loss slowly  does not change substantially, and the validation set accuracy and Mean IOU are also low.
What is the reason for this? This is because the overfitting or loss function design is unreasonable?Thanks you！
